src/twisted/web/server.py

```diff
@@ -522,6 +522,27 @@ def URLPath(self):
         return urlpath.URLPath.fromRequest(self)
 
 
+    def hyperlinkURL(self):
+        """
+        Providing hyperlink url on request object, as URLPath is
+        superceded by hyperlink.URL.
+
+        @return: a hyperlink.URL object from request prePathURL
```

Comment:
It's a bit simpler to just say `return: the URL` or something that leaves out more details here. That way, someone who doesn't even know what prePathURL is can still very likely understand what this method does.